packages/anoncreds/src/repository/AnonCredsSchemaRecord.ts

```diff
@@ -48,13 +55,20 @@ export class AnonCredsSchemaRecord extends BaseRecord<
   }
 
   public getTags() {
+    let unqualifiedSchemaId: string | undefined = undefined
+    if (this.schemaId.startsWith('did:indy:')) {
```

Comment:
I guess it's not needed for this particular case (and probably less performant), but could be more elegant to match with the regex that now has been added to anoncreds package? The same with other "startsWith('did:indy:')"